app/src/main/java/io/apicurio/registry/ccompat/dto/CompatibilityLevelDto.java

```diff
@@ -49,14 +49,7 @@ public static CompatibilityLevelDto create(Optional<CompatibilityLevel> source)
     }
 
     @JsonProperty("compatibility")
-    // Alias for compatilbility with Confluent client libraries: 
-    //.   https://docs.confluent.io/current/schema-registry/develop/api.html#get--config-(string-%20subject)
     private Level compatibilityLevel;
```

Comment:
Why not just name the field `compatibility` rather than annotate with a different name?